cfn-update-script/cfn-template-update.py

```diff
@@ -71,6 +73,18 @@ class SyncPair:
     tgt_id:str
     tgt_reg:str
 
+def check_version():
+    # Checking Python version:
+    expect_major = 3
+    expect_minor = 12
+    expect_rev = 8
+    current_version = str(version_info[0])+"."+str(version_info[1])+"."+str(version_info[2])
+    print("INFO: Script developed and tested with Python " + str(expect_major) + "." + str(expect_minor) + "." + str(expect_rev))
```

Comment:
Any >= 3.12 should work so I don't know if it's helpful to print the version including the revisions.